src/languages/es-ES.ts

```diff
@@ -3497,6 +3534,39 @@ export default class extends Language {
 		COMMAND_STAR_TOPRECEIVERS: 'Top Star Receivers',
 		COMMAND_STAR_TOPRECEIVERS_DESCRIPTION: (medal, id, stars) => `${medal}: <@${id}> (${stars} ${stars === 1 ? 'star' : 'stars'})`,
 
+		/**
+		 * ##################
+		 * SUGGESTIONS COMMANDS
+		 */
+		COMMAND_SUGGEST_NOSETUP: username => `Lo siento ${username}, pero no se ha configurado un canal de recomendaciones.`,
```

Comment:
```suggestion
		COMMAND_SUGGEST_NOSETUP: username => `Lo siento ${username}, pero no los administradores no han configurado un canal de texto para las sugerencias.`,
```

Reply:
idk how this sounds.... it's obviously a text channel, no need to mention that again.
why not just `canal para sugerencias`